### ISSUE TITLE
Nonexistent function 'get_playback_time' in base 'AnimationNodeAnimation'.

### ISSUE THREAD

**Issue by nagasawamaki** [godotengine/godot#23629]
**Godot version:**
3.1 `a16d9c6`

**OS/device including version:**
Windows 7

**Issue description:**
the function `get_playback_time` nonexist in `AnimationNodeAnimation`
error message:
`Invalid call. Nonexistent function 'get_playback_time' in base 'AnimationNodeAnimation'.`

**Comment by Naryosha** [comment]
Removed since 531dc2f. The docs will be synced soon.

**Comment by nagasawamaki** [comment]
> 
> 
> Removed since [531dc2f](https://github.com/godotengine/godot/commit/531dc2f435c1d64a2e3ef0710cc05a7c13969587). The docs will be synced soon.

Thank you.
Then how to get the playback time now?

**Comment by Naryosha** [comment]
I think it's `AnimationNodeAnimation.get_parameter("time")`. Not sure though.

**Comment by nagasawamaki** [comment]
> 
> 
> I think it's `AnimationNodeAnimation.get_parameter("time")`. Not sure though.

Tested but I got a `Nil`

**Comment by Naryosha** [comment]
Then it's probably `AnimationTreePlayer.animation_node_get_position(nodename)`

**Comment by nagasawamaki** [comment]
> 
> 
> Then it's probably `AnimationTreePlayer.animation_node_get_position(nodename)`

Thank you but my `AnimationNodeAnimation` is inside `AnimationTree` but not `AnimationTreePlayer`

**Comment by Naryosha** [comment]
Sorry, I confused those two. Actually I don't know how to use the new one.

**Comment by nagasawamaki** [comment]
Anyone knows how to get the playback time now?
Thank you!

**Comment by KoBeWi** [comment]
Seems like it should be `get_parameter("time")`, but when I try it I get error `!state is true`; same thing might've happened to the OP. Not sure what is the correct usage of this node though, maybe I'm doing something wrong. The parameter is supposed to be set only when the animation is being processed.

**Comment by TokageItLab** [comment]
I did a little investigation. It is true that `get_parameter ()` is not accessible except when processing the tree nodes. I guess that it is due to the reason why 531dc2f was merged; If multiple scenes rewrite the parameters of a single TreeNode at the same time, the process will be duplicated for as many scenes as there are. This means that processing time multiple times will increase the speed.

However, if we implement it in a way that duplicates TreeNode for each scene, the performance may be worse.

`set_playback_time()` is difficult to reimplement for the above reasons, but `get_playback_time()` may be possible. However, if it is a simple implementation, it might not be possible to get its value correctly when there are multiple scenes.

For example, let's think we have `SceneA` and it has `AnimTreeA`. If you have `B` and `C` as well, the processing order may be as follows
```
[AnimTreeA]->[AnimTreeB]->[AnimTreeC]->[SceneA]->[SceneB]->[SceneC]
```
instead of
```
[AnimTreeA]->[SceneA]->[AnimTreeB]->[SceneB]->[AnimTreeC]->[SceneC]
```

Keep in mind that **`AnimationNodeAnimation` is a resource and can be referenced by multiple scenes simultaneously**. In the first case, we can only get `AnimTreeC` values, so we need to store at least time in each scene.
